bank-sdk/example-app/src/main/java/net/gini/android/bank/sdk/exampleapp/core/DefaultServiceNetworkApi.kt

```diff
@@ -0,0 +1,90 @@
+package net.gini.android.bank.sdk.exampleapp.core
+
+import android.content.Context
+import android.text.TextUtils
+import net.gini.android.bank.api.GiniBankAPI
+import net.gini.android.bank.api.GiniBankAPIBuilder
+import net.gini.android.bank.sdk.exampleapp.R
+import net.gini.android.capture.network.GiniCaptureDefaultNetworkService
+import net.gini.android.core.api.DocumentMetadata
+
+class DefaultServiceNetworkApi (internal val context: Context, internal val logger: org.slf4j.Logger) {
```

Comment:
[Suggestion] I think the name `DefaultNetworkServicesProvider` what it does better.